### DISCUSSION TITLE
[FEATURE] List of all attributes from the catalog items

### DISCUSSION THREAD

**Discussion by gokigoks** [amzn/selling-partner-api-models#605] [Ideas]
I believe it would be very helpful to many if we had a comprehensive list of attributes offered. There is an [model](https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-model) example, but it's missing a few (e.g. publication_date, language, etc.).

Thank you for anyone that can help.

**Comment by cmlkx** [comment]
I also have this same issue.  

Running List of Missing fields (things that were present on MWS and Catalogsv0 but are NOT present on Catalogsv20220401), ASIN=1589254635 as an example:


- Author
- Page Count
- MSRP
- Binding
- PublicationDate
- ReleaseDate
- Edition


Cross post to SellerCentral/MWS Forums:
https://sellercentral.amazon.com/forums/t/sp-api-catalog-calls-missing-productgroup-specific-details/1282656

Open Case#:
10831090091

**Comment by cmlkx** [comment]
@chapmanjw is there any chance that this could be looked at, or that the fields that are present in v0 can be preserved?

**Comment by cmlkx** [comment]
@sungolivia  Any chance that this issue can be tackled?  We rely heavily on these details to verify we are matching the correct item!

**Comment by sungolivia** [comment]
this sounds similar to this [discussion](https://github.com/amzn/selling-partner-api-docs/discussions/1609) thread going on, let me know otherwise but I will monitor both and see where feature is at!

**Reply by gokigoks** [reply]
@sungolivia It is not. What we need is a documented list of all possible fields the API can return.

**Comment by cmlkx** [comment]
@sungolivia  Any updates on this?  We are getting close to the end of the year and it's critical

**Comment by cmlkx** [comment]
@sungolivia @chapmanjw  Just got the 30 day notice.  We would apricate some kind of update on this issue.
